codecov_auth/admin.py

```diff
@@ -49,18 +49,6 @@ def impersonate_owner(self, request, queryset):
 impersonate_owner.short_description = "Impersonate the selected owner"
 
 
```

Comment:
Remove the owner profile section in admin page as there's nothing left without agreements, instead add it to user form